pre_commit_hooks/make/phony-targets.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+EXITCODE=0
+
+validate_phony_targets() {
+    local source=$1
+    local targets=$(sed -n 's#^.PHONY:\(.*\)#\1#p' <<< $source)
```

Comment:
```suggestion
    local file=$1
    local source=$(make -npq -f "${file}" | grep -E '^.PHONY:')
    local targets=$(sed -n 's#^.PHONY:\(.*\)#\1#p' <<< "${source}")
```

Reply:
doesn't work in OSX and just prints the whole file

Reply:
great idea! resolved in b474bbb